google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -307,6 +309,11 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      // Called on New Project or New Module dialogs.
+      UsageTrackerProvider.getInstance()
+          .trackEvent(GctTracking.APP_ENGINE_ADD_SUPPORT)
```

Comment:
This is called on new projects, new modules and Add Framework Support, which is why I opted for a more generic name.